purchase_request_to_rfq/models/purchase_request.py

```diff
@@ -0,0 +1,146 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016 Eficent Business and IT Consulting Services S.L.
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl-3.0).
+
+from datetime import datetime
+
+from dateutil.relativedelta import relativedelta
+
+from openerp import _, api, exceptions, fields, models
+
+_PURCHASE_ORDER_LINE_STATE = [
```

Comment:
Don't use this. Make a selection attribute. You can query later this list making `self.env['model_name']._fields['field_name'].selection`